Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -482,6 +489,10 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 NodejsPackage.Instance.CheckSurveyNews(false);
                 ModulesNode.ReloadHierarchySafe();
 
+#if DEV14
+                TryToAcquireTypings(new[] { "node" });
```

Comment:
Could you verify that line *always* gets called in Reload first? Would be more performant to call tsd just once if possible.